src/APIM_ARMTemplate/apimtemplate/Common/Templates/Abstractions/Template.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using Newtonsoft.Json;
 using System.Collections.Generic;
 
-namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common
+namespace apimtemplate.Common.Templates.Abstractions
 {
     public class Template
```

Comment:
Let's create a file per class

Reply:
created separate files for all abstract templates